include/shellstrings.h

```diff
@@ -0,0 +1,128 @@
+
+/* 
+ * String output functions to be used in the shell
+*/
+
+#ifndef INCLUDE_SHELLSTRINGS_H_
+#define INCLUDE_SHELLSTRINGS_H_
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/*
+ * shell_prompt - Default text input prompt string for the shell.
+ *
+ * Parameters:
+ *  - None.
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_prompt();
+
+/*
+ * shell_intro - Introduction text for the shell.
+ *
+ * Parameters:
+ *  - None.
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_intro();
+
+/*
+ * shell_save - Modification saving text for the shell.
+ *
+ * Parameters:
+ *  - None.
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_save();
+
+/*
+ * shell_print - Print edited text for viewing
+ *
+ * Parameters:
+ * 	- None.
+ *
+ * Returns:
+ * 	- None.
+ */
+ void shell_print(char** lines);
+
+
+/*
+ * shell_error - Default error prompt string for the shell.
+ *
+ * Parameters:
+ *  - error_text: String for the specific error text.
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_error(char* error_text);
+
+/*
+ * shell_input - Input file confirmation prompt for the shell.
+ *
+ * Parameters:
+ *  - input_file: String of the file input by the user.
+ *  - mode: String for the mode being used (either quiet or verbose) 
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_input(char* input_file, char* mode);
+
+/*
+ * shell_help - Default text input prompt string for the shell.
+ *
+ * Parameters:
+ *  - None.
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_help();
+
+/*
+ * shell_outro - Outro text for the shell.
```

Comment:
Do we need outro text? Most interactive tools just exit when you `Ctrl-D`.

Reply:
Deleted. @firatciftci  note